### PR TITLE
Readds Laser carbines.

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Adds back in the Laser carbine

:cl: optional name here
fix: missing hopes and dreams
code: orgized the weapon file to be more cat brain friendly
/:cl:

[why]
Kev didnt mean to shadow realm the laser carbine.
I miss you carbine!

### CODE REVIEW

**Comment by Poojawa** [comment]
https://github.com/Citadel-Station-13/Citadel-Station-13/blob/master/modular_citadel/code/modules/projectiles/guns/energy/laser.dm#L21

this already exists

**Comment by silicons** [comment]
I'm really concerned about the balance of adding yet another magrifle, even if it has 1/2.5 of the DPS.

File: modular_citadel/code/modules/projectiles/guns/energy/laser.dm [hunk 1]
```diff
@@ -17,30 +17,3 @@
 /obj/item/gun/energy/laser/redtag
 	lefthand_file = 'icons/mob/inhands/weapons/guns_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/guns_righthand.dmi'
-	
-/obj/item/gun/energy/laser/carbine
```
Comment (Poojawa) [inline modular_citadel/code/modules/projectiles/guns/energy/laser.dm:21]: by 'this already exists' I mean your addition is superfluous

Comment (Trilbyspaceclone) [reply modular_citadel/code/modules/projectiles/guns/energy/laser.dm:21]: I meant to move it to non-modular files

**Review by silicons — APPROVED** [review]

**Pull request merged**